crates/pallet-evm-balances/src/tests/fungible_conformance_tests.rs

```diff
@@ -0,0 +1,66 @@
+use frame_support::traits::fungible::{conformance_tests, Inspect, Mutate};
+use paste::paste;
+
+use crate::{mock::*, *};
+
+macro_rules! run_tests {
```

Comment:
Can't we import this macro from the `conformance_tests::inspect_mutate`?

Reply:
We can't:
- it's not there but in a different place
- it's not expored
- it's not generic enough to be reusable like this